### PR TITLE
Added VACOLS deep monitoring 

### PR DESCRIPTION
**Pull request by askldjd**
See full details
https://github.com/department-of-veterans-affairs/appeals-deployment/issues/740

### CODE REVIEW

File: app/services/vacols_service.rb [hunk 1]
```diff
@@ -38,16 +38,100 @@ def query_service
...
+        name: 'sum_all_db_time_24hrs'
+      }, sum_all_db_time_24hrs[0]['dbtime'])
+
+      # Summing Caseflow DB Time from ASH table
+      query = <<-SQL
+        select count(*) DBTime
+        from v$active_session_history
+        where sample_time > sysdate - 1
+          and session_type <> 'BACKGROUND'
+          and v$active_session_history.user_id = dsuser_id
```
Comment (kierachell) [inline app/services/vacols_service.rb]: Would that count this query too? Can probably filter by `machine`?

File: app/services/vacols_service.rb [hunk 2]
```diff
@@ -38,16 +38,100 @@ def query_service
...
+          name: stm['stat_name']
+        }, stm['time'])
+      end
+
+
+      # Summing DB Time from ASH table
+      query = <<-SQL
+        select count(*) DBTime
+        from v$active_session_history
+        where sample_time > sysdate - 1
```
Comment (kierachell) [inline app/services/vacols_service.rb]: Do you want the whole day or just the past hour? What is the expected rate of change?

File: app/services/vacols_service.rb [hunk 3]
```diff
@@ -9,7 +9,7 @@ def initialize
     @name = @@service_name
     @service = "VACOLS"
     @env = ENV['VACOLS_HOST']
-    @api = "VACOLS.BRIEFF"
+    @api = "BRIEFF and ASH"
```
Comment (kierachell) [inline app/services/vacols_service.rb]: Or maybe split them up to VACOLS.BRIEFF and VACOLS.ASH services?

Comment (askldjd) [inline app/services/vacols_service.rb]: very good point. I will do that.

File: app/services/vacols_service.rb (hunk 2 above)
Comment (askldjd) [inline app/services/vacols_service.rb]: I don't think it matters whether it is the whole day or last hour. The delta is what matters.

The sampling freq is 1s according to https://www.youtube.com/watch?v=Aknb_iZrPbc

**Pull request merged**